api/tests/admin/custom_views/venue_view_test.py

```diff
@@ -432,6 +436,68 @@ def test_reindex_when_tags_updated(
         mocked_async_index_venue_ids.assert_not_called()
         mocked_async_index_offers_of_venue_ids.assert_not_called()
 
+    @clean_database
+    @patch("wtforms.csrf.session.SessionCSRF.validate_csrf_token")
+    def test_venue_user_receives_permanent_venue_email(self, mocked_validate_csrf_token, app):
+        AdminFactory(email="user@example.com")
+        venue = offerers_factories.VenueFactory(isPermanent=False)
+
+        data = dict(
+            name=venue.name,
+            siret=venue.siret,
+            city=venue.city,
+            postalCode=venue.postalCode,
+            address=venue.address,
+            publicName=venue.publicName,
+            latitude=venue.latitude,
+            longitude=venue.longitude,
+            isPermanent=True,
+            adageId=venue.adageId,
+        )
+
+        client = TestClient(app.test_client()).with_session_auth("user@example.com")
+        response = client.post(f"/pc/back-office/venue/edit/?id={venue.id}", form=data)
+
+        assert response.status_code == 302
+
+        venue = Venue.query.get(venue.id)
+        assert venue.isPermanent
+        assert len(mails_testing.outbox) == 1
+        assert mails_testing.outbox[0].sent_data["params"] == {
+            "VENUE_FORM_URL": build_pc_pro_venue_link(venue),
+            "VENUE_NAME": venue.publicName or venue.name,
+        }
+        assert mails_testing.outbox[0].sent_data["template"] == asdict(TransactionalEmail.VENUE_NEEDS_PICTURE.value)
+
+    @clean_database
+    @patch("wtforms.csrf.session.SessionCSRF.validate_csrf_token")
+    @pytest.mark.parametrize("prev_is_permanent, new_is_permanent", [(True, True), (False, False), (False, True)])
```

Comment:
As-tu lancé les tests ? Il faut aussi utiliser ces variables :-)
```
def test_venue_user_does_not_receives_permanent_venue_email_if_disabled_or_not_changed(
        self, mocked_validate_csrf_token, app, prev_is_permanent, new_is_permanent 
    ):
```
puis remplacer `True` et `False` dans le test, et un assert à la fin :
` assert venue.isPermanent is new_is_permanent`

Reply:
Oui, j'ai juste oublié de pousser la suite. Je viens de le faier.

Reply:
assert venue.isPermanent is new_is_permanent